lib/jit_preloader/active_record/base.rb

```diff
@@ -101,6 +101,13 @@ def has_many_aggregate(assoc, name, aggregate, field, table_alias_name: nil, def
           if jit_preloader
             reflection = association(assoc).reflection
             primary_ids = jit_preloader.records.collect{|r| r[reflection.active_record_primary_key] }
+            dive_limit = dive_limit || JitPreloader.query_dive_limit
+            if dive_limit
+              slices = primary_ids.each_slice(dive_limit)
```

Comment:
If we're setting `dive_limit` to, say, 3, we're going to be looking at sets of size 2.  That doesn't make a ton of sense to me => shouldn't the limit be inclusive and not exclusive?

Reply:
After discussing this with @fimmtiu, I will change `dive_limit` to `max_ids_per_query ` and remove the `minus one` logic. I will update README later to explain better how and when users should use this. That should resolve your concern, hopefully.